Assets/Scripts/UI/InGame/Graphics/GraphicsController.cs

```diff
@@ -1,173 +1,125 @@
-﻿using UnityEngine;
+﻿using System;
+using UnityEngine;
 using UnityEngine.UI;
-using System;
 
 namespace Assets.Scripts.UI.InGame
 {
-	public class GraphicsController : MonoBehaviour {
+    public class GraphicsController : MonoBehaviour {
```

Comment:
Indentation is also messed up here. So same as other comment:
`The indentation of this class seems messed up somehow. So please format the document (Visual Studio: Ctrl+K, D )`

Reply:
looked fine on my end, still formatted tho